mne/preprocessing/tests/test_ica.py

```diff
@@ -271,6 +272,11 @@ def test_ica_additional():
     with warnings.catch_warnings(record=True):
         ica.fit(raw, picks=None, start=start, stop=stop2)
 
+    # test corrmap
+    ica2 = deepcopy(ica)
+    corrmap([ica, ica2], (0, 0), threshold=0.99, inplace=True)
+    assert_true(ica.labels["bads"] == ica2.labels["bads"])
```

Comment:
#IMO: arbitrary. There is no agreed-upon terminology for IC clusters‚ nor an exhaustive list, nor any fixed rule of what to do with them. "bads" is just a default, but maybe you don't want to find bad ones, but e.g. left mu?